test/integration/elasticquota_controller_test.go

```diff
@@ -56,6 +59,7 @@ func TestElasticController(t *testing.T) {
 	s := scheme.Scheme
 	runtime.Must(schedv1alpha1.AddToScheme(s))
 
+	log.SetLogger(klogr.New())
```

Comment:
Are these changes mandatory? If not, let's remove them; if yes, how about deferring to the PR related with elasticquota?

Reply:
I added while i was debugging the integration tests, deleted already.